docs/source/r-space.rst

```diff
@@ -63,6 +63,27 @@ The corresponding Python code is:
   exact.interpolate(y - 0.5)
   print sqrt(assemble((u - exact)*(u - exact)*dx))
 
+Setting and retrieving the value of a function in `R`
+-----------------------------------------------------
+
+Functions in the space `R` are equivalent to a single floating point value. The
```

Comment:
Do you mean \`R\` (i.e. "interpreted text, left up to the processor") or do you mean \`\`R\`\` (\texttt{R})

Reply:
I mean `R`, which is defined to be `$R$` by setting math as the default role at the top of the file. This matches the rest of the file.